### PR TITLE
Add instances for conv_scale with fp8@bf8->fp8

### PR DESCRIPTION
**Pull request by geyyer**
- Update device op api
- Add an example
- Add instances
- Add a profiler mode
- Add a client example

### CODE REVIEW

File: library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp [hunk 1]
```diff
@@ -1156,36 +1177,44 @@ struct DeviceOperationInstanceFactory<ck::tensor_operation::device::DeviceGroupe
...
-                         is_same_v<OutDataType, ck::f8_t> && is_same_v<ComputeType, ck::f8_t>)
+                         is_same_v<OutDataType, ck::f8_t> && is_same_v<AComputeType, ck::f8_t>)
             {
                 add_device_grouped_conv3d_fwd_xdl_ndhwgc_gkzyxc_ndhwgk_f8_instances(op_ptrs);
             }
 #endif
 #ifdef CK_ENABLE_BF8
             if constexpr(is_same_v<InDataType, ck::bf8_t> && is_same_v<WeiDataType, ck::bf8_t> &&
-                         is_same_v<OutDataType, ck::f8_t> && is_same_v<ComputeType, ck::bf8_t>)
+                         is_same_v<OutDataType, ck::f8_t> && is_same_v<AComputeType, ck::bf8_t>)
```
Comment (bartekxk) [inline library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp]: Here and above: Maybe should we add a check for BComputeType?

**Review by bartekxk — COMMENTED** [review]
LGTM, just one comment

File: library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp (hunk 1 above)
Comment (geyyer) [reply library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp]: @bartekxk, thanks, good point, I'll add a BComputeType check!

Comment (geyyer) [reply library/include/ck/library/tensor_operation_instance/gpu/grouped_convolution_forward.hpp]: Updated in [4319dd9](https://github.com/ROCm/composable_kernel/pull/1220/commits/4319dd99288cc7f4b19158d24c1c462f1fe7757f)

**Review by bartekxk — DISMISSED** [review]

**Review by bartekxk — APPROVED** [review]

**Review by zjing14 — APPROVED** [review]

**Pull request merged**